ansible/roles/debops.gitlab/templates/etc/default/gitlab.j2

```diff
@@ -43,3 +43,7 @@ gitlab_pages_dir="{{ gitlab_pages_checkout }}"
 gitlab_pages_pid_path="$pid_path/gitlab-pages.pid"
 gitlab_pages_log="$app_root/log/gitlab-pages.log"
 gitlab_pages_options="-pages-domain {{ gitlab_pages_domain }} -pages-root {{ gitlab_pages_path }} -listen-proxy 127.0.0.1:{{ gitlab_pages_port }}"
+
+gitaly_dir=$(cd $app_root/../gitaly && pwd)
```

Comment:
Hmm, that's unusual... But it looks like [the upstream does it this way as well](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/lib/support/init.d/gitlab.default.example#L90) so I guess we can use it. And this file is not touched by `systemd` at all.

I think that you should add
```
gitaly_enabled=true
```
for completeness and parity with upstream.

Reply:
@drybjed Uhm, I never noticed we support SystemV init scripts. Do we even want that?